utility/validation.py

```diff
@@ -302,268 +302,399 @@ def date_and_time_validation(date_time, format):
             r"^(([A-Za-z]+)[- /.,](0?[1-9]|[12][0-9]|3[01])[- /.,]([2-9][0-9]|(19))\s?((0?[0-9]|1[0-9]|2[0-4])[:](0?[0-9]|[1-5][0-9])\s?(AM|am|PM|pm|Am|aM|Pm|pM)))$",
             date_time)  # mon-dd-yy HH:MM AM/PM
 
-        if True:
-            if date_format == 'dd-mm-yyyy hh:mm' or date_format == 'dd.mm.yyyy hh:mm' or date_format == 'dd mm yyyy hh:mm' or date_format == 'dd/mm/yyyy hh:mm':
-                if x:
-                    var, date_time = Validate._24hr(x)
-                    date, month, year = re.split(r"[\s -/.]", var)
-                    curr_format = Validate.datething(date, month, year, date_time)
-                    return curr_format
-
-                else:
-                    print("Format does not match with the entered date and time.")
-
-            elif date_format == 'mm-dd-yyyy hh:mm' or date_format == 'mm.dd.yyyy hh:mm' or date_format == 'mm dd yyyy hh:mm' or date_format == 'mm/dd/yyyy hh:mm':
-                if y:
-                    var, date_time = Validate._24hr(y)
-                    month, date, year = re.split(r"[\s -/.]", var)
-                    curr_format = Validate.datething(date, month, year, date_time)
-                    return curr_format
-                else:
-                    print("Format does not match with the entered date and time.")
-
-            elif date_format == 'mm-dd-yy hh:mm' or date_format == 'mm.dd.yy hh:mm' or date_format == 'mm dd yy hh:mm' or date_format == 'mm/dd/yy hh:mm':
-                if z:
-                    var, date_time = Validate._24hr(z)
-                    month, date, year = re.split(r"[\s -/.]", var)
-                    curr_format = Validate.datething(date, month, int(year) + 2000, date_time)
-                    return curr_format
-                else:
-                    print("Format does not match with the entered date and time.")
-
-            elif date_format == 'dd-mm-yy hh:mm' or date_format == 'dd.mm.yy hh:mm' or date_format == 'dd mm yy hh:mm' or date_format == 'dd/mm/yy hh:mm':
-                if a:
-                    var, date_time = Validate._24hr(a)
-                    date, month, year = re.split(r"[\s -/.]", var)
-                    curr_format = Validate.datething(date, month, int(year) + 2000, date_time)
```

Comment:
Function Validate.date_and_time_validation refactored with the following changes:

Remove redundant conditional ([remove-redundant-if](https://docs.sourcery.ai/refactorings/remove-redundant-if/))
Replace multiple comparisons of same variable with in operator ([merge-comparisons](https://docs.sourcery.ai/refactorings/merge-comparisons/))

Reply:
The comment doesn't use bullet points